tests/test_ansprechpartner.py

```diff
@@ -49,6 +49,7 @@ def test_de_serialisation_minimal_attributes(self):
         json_string = schema.dumps(ansprechpartner, ensure_ascii=False)
         assert "Müller-Schmidt" in json_string
         assert "Mühlenweg" in json_string
+        assert '"FRAU"' in json_string
```

Comment:
I thought there should be just `FRAU` without the quotes.

Reply:
nope. because to json, the enum just behaves like a normal string (or int). only when the json is deserialized into something type it becomes an enum value.